src/backend/distributed/executor/multi_utility.c

```diff
@@ -31,6 +31,7 @@
 #include "distributed/worker_manager.h"
 #include "distributed/worker_protocol.h"
 #include "foreign/foreign.h"
+#include "nodes/print.h"
```

Comment:
You are nothing if not observant. It was here for a `elog_node_display` and I forgot to remove it. Saw it last night but didn't feel like rewriting the commit again for one line. Will fix.